src/modules/physics/box2d/wrap_Physics.cpp

```diff
@@ -477,12 +477,93 @@ int w_setMeter(lua_State *L)
 	return 0;
 
 }
+
 int w_getMeter(lua_State *L)
 {
 	lua_pushinteger(L, Physics::getMeter());
 	return 1;
 }
 
+int w_computeLinearStiffness(lua_State *L) {
```

Comment:
Brace style (here and the other functions below)

Reply:
I've become too reliant on auto-formatters :marge: